### PR TITLE
Make Poll options more clear

### PR DESCRIPTION
**Pull request by MrLar**
The current help embed for createpoll doesnt really tell you the format for the options parameter. As well as it doesnt state that admins can cancel any poll.

### CODE REVIEW

File: src/main/java/net/kodehawa/mantarobot/commands/MiscCmds.java [hunk 1]
```diff
@@ -324,10 +324,10 @@ public MessageEmbed help(GuildMessageReceivedEvent event) {
                 return helpEmbed(event, "Poll Command")
                         .setDescription("**Creates a poll**")
                         .addField("Usage", "`~>poll [-options <options>] [-time <time>] [-name <name>]`", false)
-                        .addField("Parameters", "`-options` The options to add. Minimum is 2 and maximum is 9.\n" +
+                        .addField("Parameters", "`-options` The options to add. Minimum is 2 and maximum is 9. For instance: `Pizza,Spaghetti,Pasta`.\n" +
                                 "`-time` The time the operation is gonna take. The format is as follows `1ms29s` for 1 minute and 21 seconds. Maximum poll runtime is 45 minutes.\n" +
                                 "`-name` The name of the poll for reference.", false)
-                        .addField("Considerations", "To cancel the running poll type &cancelpoll. Only the person who started it can cancel it.", false)
+                        .addField("Considerations", "To cancel the running poll type &cancelpoll. Only the person who started or an Admin can it can cancel it.", false)
                         .build();
```
Comment (Kodehawa) [inline src/main/java/net/kodehawa/mantarobot/commands/MiscCmds.java]: "or an admin can it can"

Comment (MrLar) [reply src/main/java/net/kodehawa/mantarobot/commands/MiscCmds.java]: woops

**Comment by Kodehawa** [comment]
go sleep or :blobb1nzy:

**Comment by MrLar** [comment]
at least merge it uwu

File: src/main/java/net/kodehawa/mantarobot/commands/MiscCmds.java [hunk 2]
```diff
@@ -324,10 +324,10 @@ public MessageEmbed help(GuildMessageReceivedEvent event) {
                 return helpEmbed(event, "Poll Command")
                         .setDescription("**Creates a poll**")
                         .addField("Usage", "`~>poll [-options <options>] [-time <time>] [-name <name>]`", false)
-                        .addField("Parameters", "`-options` The options to add. Minimum is 2 and maximum is 9.\n" +
+                        .addField("Parameters", "`-options` The options to add. Minimum is 2 and maximum is 9. For instance: `Pizza,Spaghetti,Pasta`.\n" +
```
Comment (Kodehawa) [inline src/main/java/net/kodehawa/mantarobot/commands/MiscCmds.java]: Maybe add an example with spaces?

**Comment by MrLar** [comment]
<-<

**Comment by Kodehawa** [comment]
Good boy :3

**Pull request merged**